src/main/java/command/Command.java

```diff
@@ -0,0 +1,31 @@
+package command;
+
+import util.DukeException;
+import util.Parser;
+
+/**
+ * Abstract class that executes the command given by the user.
+ */
+public abstract class Command {
```

Comment:
I like that you made this an abstract class to ensure that a `Command` cannot be instantiated! 💯